sqldelight-gradle-plugin/src/test/fixtures/recursive-view/failure.txt

```diff
@@ -0,0 +1,6 @@
+Table.sq line 2:12 - Recursive subquery found in view view1
```

Comment:
didn't even think of that. I absolutely can because I use a stack to keep track.